mixing/field.go

```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2024 The Decred developers
```

Comment:
This is https://github.com/decred/cspp/blob/master/dcnet/field.go

`// Copyright (c) 2019-2024 The Decred developers`